stream-chat-android-client/src/test/java/io/getstream/chat/android/client/EventChatJsonProvider.kt

```diff
@@ -656,3 +656,15 @@ private fun createReactionJsonString() =
             "created_at": "2020-06-29T06:14:28.000Z"
         }
     """.trimIndent()
+
+internal fun createMarkAllReadEventStringJson() =
+    """
+        {
+           "unread_count":0.0,
+           "unread_channels":0.0,
+           "total_unread_count":0.0,
```

Comment:
Why are they double values instead of integer?

Reply:
Copied directly from a real event's json payload

Reply:
if i remember correctly the json spec doesn't differentiate between float and ints